### ISSUE TITLE
Can not download torrent

### ISSUE THREAD

**Issue by SNBogdanov** [transmission/transmission#6924]
### What is the issue?

On windows with transmission-daemon 4+ I found strange behaviour. I added torrent and it shows a lot of seeders but all of them are in "d" state, so no download ever started.
But as soon as I restarted daemon service all these seeders started to send me data. And this is what I wanted.
Such behaviour is for 80% of torrents I have.
Some of torrents manage to start downloading.
My computer is behind NAT with no public port opened, but this can not be issue since after restart torrent managed to start download.

### Which application of Transmission?

transmission-daemon

### Which version of Transmission?

4.0.6

**Comment by kiwixz** [comment]
`d` flag means the peer is choking you (deciding not to upload to you because they have better others peers).
You should try increasing your upload speed limit if you have any.

**Comment by SNBogdanov** [comment]
> `d` flag means the peer is choking you (deciding not to upload to you because they have better others peers). You should try increasing your upload speed limit if you have any.

So why it does not choke me after restart daemon?

**Comment by kiwixz** [comment]
Because when you reconnect they *may* give you time to see if you'll upload to them fast enough.
And I you don't they'll choke you again.

**Comment by SNBogdanov** [comment]
No. If I connected after restart I keep connected till the end of dowload. And remove upload limit does not solve the problem since nobody can connect to me to get data. After restart all connections, up and down restored and workibg.
